repose-aggregator/components/filters/keystone-v2/src/main/resources/META-INF/schema/config/keystone-v2.xsd

```diff
@@ -0,0 +1,383 @@
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="region" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Region to match on the user's service catalog entry
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="name" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Name of the service to match in the user's service catalog entry
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="type" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Type to match in the user's service catalog entry
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+
+    <xs:complexType name="CacheType">
+        <xs:sequence>
+            <xs:element name="timeouts" type="CacheTimeoutsType" minOccurs="0" maxOccurs="1"/>
+            <!-- todo: xs:element name="eviction-feeds" type="AtomFeeds" minOccurs="0" maxOccurs="1"/-->
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="CacheTimeoutsType">
+        <xs:sequence>
+            <xs:element name="token" type="ZeroOrPositiveInteger" default="600" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="group" type="ZeroOrPositiveInteger" default="600" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="endpoints" type="ZeroOrPositiveInteger" default="600" minOccurs="0" maxOccurs="1"/>
+            <!-- todo: xs:element name="user" type="ZeroOrPositiveInteger" default="600" minOccurs="0" maxOccurs="1"/-->
+        </xs:sequence>
+        <xs:attribute name="variability" type="ZeroOrPositiveInteger" default="0" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        This value will be added or subtracted to the cache timeouts to make sure that the cached items
+                        have some variability so they don't all expire at the exact same time. (Defaults to no
+                        variability)
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:complexType name="DelegatingType">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Whether or not you would like this filter to populate the delegation headers. Inclusion means
+                    you do.
```

Comment:
Should probably also mention that it won't reject requests that it ordinarily would.